app/views/provider_interface/decisions/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :browser_title, title_with_error_prefix(t('.title'), @wizard.errors.any?) %>
+<% content_for :before_content, govuk_back_link_to(provider_interface_application_choice_path(@application_choice)) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">
+      <span class="govuk-caption-l"><%= @application_choice.application_form.full_name %></span>
+      <%= t('.title') %>
+    </h1>
+    <%= render ProviderInterface::CourseSummaryComponent.new(course_option: @application_choice.course_option) %>
+
+    <%= form_with model: @wizard, url: provider_interface_application_choice_decision_path(@application_choice), method: :post do |f| %>
+      <%= f.govuk_error_summary %>
```

Comment:
I wonder if this should be at the top of the summary component? 🤔

Reply:
Yes I think it should as that's the case on the interview page as well.